benchmark/compute_accuracy.py

```diff
@@ -0,0 +1,182 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2015--, The WGS-HGT Development Team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+"""
+Given known HGTs, compute precision, recall and F-score for various tools
+=========================================================================
+"""
+
+import sys
+import click
+import re
+
+
+def parse_expected_transfers(ground_truth_f):
+    """ Parse ALF's log file.
+
+    Parameters
+    ----------
+    ground_truth_f: string
+        file descriptor to ALF's log file
+
+    Returns
+    -------
+    expected_transfers: list of tuples
+        list of transfers with each tuple representing (organism donor, gene
+        donated, organism recipient, gene received)
+
+    Notes
+    -----
+    Report horizontal gene transfers with information on organism donor,
+    gene donated, organism recipient and gene received.
+    """
+    expected_transfers = []
+    string = "lgt from organism "
```

Comment:
I'm not sure about getting messy for 3 options, dozens I would agree. They could be dicts too, it it makes it clearer.